pkg/scheduler/preemption/preemption.go

```diff
@@ -71,6 +72,10 @@ type preemptionCtx struct {
 	requests          resources.FlavorResourceQuantities
 	tasRequests       cache.WorkloadTASRequests
 	frsNeedPreemption sets.Set[resources.FlavorResource]
+
+	// tasResult is used to propagate the result from the successful
+	// workloadFits up, so that it can be used to update the assignment.
+	tasResult *cache.TASAssignmentsResult
```

Comment:
```suggestion

	tasRequests cache.WorkloadTASRequests
	// tasResult is used to propagate the result from the successful
	// workloadFits up, so that it can be used to update the assignment.
	tasResult *cache.TASAssignmentsResult
```

If this new line indicates TAS related grouping, it would be better to move `tasRequests` to here.

Reply:
We could move it next to tasRequests because both are related to TAS. OTOH, I was thinking about putting it separately, because this is the only field which is set during the algorithm, which makes is special. WDYT?

Reply:
Rafactored in https://github.com/kubernetes-sigs/kueue/commit/f1a3f568c6a062fc5c4491e918957574fb032fa6 which drops the field entirely.